### DISCUSSION TITLE
Would eclipse-edc remain in Github or would there be a federated copy/migration in Gitlab

### DISCUSSION THREAD

**Discussion by justinanthonyvtt** [eclipse-edc/.github#66] [Q&A]
There seems to be projects that utilise the Gitlab code repository set up by the Eclipse foundation at :
https://gitlab.eclipse.org/eclipse/

for example:
https://gitlab.eclipse.org/eclipse/xfsc
Are they any discussion on moving/migrating the repository or perhaps not?

**Comment by mspiekermann** [comment]
It's not planned to move or migate the project to GitLab.
